announcement/announcement-tool/tool/src/webapp/vm/announcement/chef_announcements-revise.vm

```diff
@@ -182,7 +182,7 @@ function resizeFrame(){
 							<label for="annc-highl-key">
 								<label class="annc-highl-label" for="highlight" name="highlight-key">
 									#set($isHightlighted = $message.getProperties().getProperty("highlight") == "true")
-									<input type="checkbox" id="highlight" #if ($isHightlighted) checked="" #end name="highlight" value="true">
+									<input type="checkbox" id="highlight" #if ($isHightlighted || $highlight) checked="" #end name="highlight" value="true">
```

Comment:
Do you think makes more sense to add it to the $isHightlighted variable?

#set($isHightlighted = $message.getProperties().getProperty("highlight") == "true" || $highlight )
<input type="checkbox" id="highlight" #if ($isHightlighted) checked="" #end name="highlight" value="true">